app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/settings/SettingsExampleFragment.java

```diff
@@ -100,6 +100,7 @@ public void onAttach(Context context) {
 
         @Override
         public void onDetach() {
```

Comment:
onAttach is called before,  so there is no reason that fragments.size() is 0.
The real reason is onAttach(Context context) is after api23,  so before api23,  it should override onAttach(Activity)